aspnetcore/includes/net-prereqs-vsc-9.0.md

```diff
@@ -0,0 +1,5 @@
+* [Visual Studio Code](https://code.visualstudio.com/download)
+* [C# Dev Kit for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit)
+* [!INCLUDE [.NET 9.0 SDK](~/includes/9.0-SDK.md)]
+
+The Visual Studio Code instructions use the .NET CLI for ASP.NET Core development functions such as project creation. You can follow these instructions on macOS, Linux, or Windows and with any code editor. Minor changes may be required if you use an integrated development environment (IDE) other than Visual Studio Code.
```

Comment:
Some of this is no longer true when C# Dev Kit is used, and this article includes C# Dev Kit in prereqs.

```suggestion
You can follow the Visual Studio Code instructions on macOS, Linux, or Windows. Changes may be required if you use an integrated development environment (IDE) other than Visual Studio Code.
```